python/paddle/fluid/tests/unittests/test_while_loop_op.py

```diff
@@ -41,9 +41,7 @@ def body(i):
             ten = layers.fill_constant(shape=[1], dtype='int64', value=10)
             out = layers.while_loop(cond, body, (i, ))
 
-        place = fluid.CUDAPlace(0) if core.is_compiled_with_cuda(
-        ) else fluid.CPUPlace()
-        exe = fluid.Executor(place)
+        exe = fluid.Executor()
```

Comment:
It's fine here .... but better choice is putting it in executor related tests ....

Reply:
Done.